### PR TITLE
Fixed copyright date

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
- fixed https://github.com/mitodl/micromasters/issues/2333

@pdpinch 

screen shot 2017-01-18 at 3 55 13 pm

### CODE REVIEW

File: ui/templates/footer.html [hunk 1]
```diff
@@ -29,7 +29,8 @@
             </a>
           </div>
         {% endif %}
-         <div class="footer-copy">&copy; 2016, Massachusetts Institute of Technology</div>
+         {% now "Y" as current_year %}
+         <div class="footer-copy">&copy; {{ current_year|add:"-1" }}-{{ current_year }}, Massachusetts Institute of Technology</div>
```
Comment (pdpinch) [inline ui/templates/footer.html]: The starting year should be fixed at 2016, not calculated at the current year minus 1.

Comment (pdpinch) [inline ui/templates/footer.html]: Also, I don't know why there's a comma between the year and `Massachusetts`. That should be removed.

**Comment by amir-qayyum-khan** [comment]
Done @pdpinch

File: ui/templates/footer.html [hunk 2]
```diff
@@ -29,7 +29,7 @@
             </a>
           </div>
         {% endif %}
-         <div class="footer-copy">&copy; 2016, Massachusetts Institute of Technology</div>
+         <div class="footer-copy">&copy; 2016-{% now "Y" %}, Massachusetts Institute of Technology</div>
```
Comment (pdpinch) [inline ui/templates/footer.html]: Please remove the comma between the year and `Massachusetts`

**Comment by amir-qayyum-khan** [comment]
Done @pdpinch

**Comment by pdpinch** [comment]
👍

**Pull request merged**